.github/workflows/build-graphscope-wheels-linux.yml

```diff
@@ -79,6 +129,27 @@ jobs:
         cd ${GITHUB_WORKSPACE}
         sudo make graphscope-image
 
+    - name: Extract Tag Name
+      id: tag
+      env:
+         GITHUB_REF_DUMMY: refs/tags/v0.9.1
```

Comment:
v0.9.1 looks strange here. Could it be "nightly" or something like that?

Reply:
Just for test release workflow